src/parsers/markdown.mjs

```diff
@@ -58,4 +70,35 @@ const createParser = linter => {
   return { parseApiDocs, parseApiDoc };
 };
 
+/**
+ * Retrieves all Node.js major versions from the provided CHANGELOG.md file
+ * and returns an array of objects containing the version and LTS status.
+ * @param {string|URL} path Path to changelog
+ * @returns {Promise<Array<ApiDocReleaseEntry>>}
+ */
+export const parseChangelog = async path => {
+  const changelog = await loadFromURL(path);
+
+  const nodeMajors = Array.from(changelog.matchAll(NODE_VERSIONS_REGEX));
```

Comment:
This requires a changelog file to be passed no? Wouldn't this require the changelog argument to be mandatory?

Reply:
`changelog` is defined on line 80